main.py

```diff
@@ -82,7 +83,7 @@ def checktodo_main():
             elif current_file is not None \
                     and current_pos is not None \
                     and line.startswith('+'):
-                if 'todo' in line.lower():
+                if re.match(r'[^\w\d]todo(?![\w\d])', line.lower()):
```

Comment:
What do you think about using `([^\w\d]|^)todo(?![\w\d])`? It will cover an edge case where `todo` is the first and only word in a file:
<img width="269" alt="Screenshot 2020-02-08 at 13 50 51" src="https://user-images.githubusercontent.com/747087/74081907-0b0ba700-4a7a-11ea-98b7-324419b9da6f.png">

Reply:
Good suggestion!